libfwupdplugin/fu-usb-device.c

```diff
@@ -525,6 +526,7 @@ fu_usb_device_open(FuDevice *device, GError **error)
 {
 	FuUsbDevice *self = FU_USB_DEVICE(device);
 	FuUsbDevicePrivate *priv = GET_PRIVATE(self);
+        g_autoptr(GError) error_local = NULL;
```

Comment:
spaces? Can you run clang-format.sh on this please? Also, if you set up precommit (run ./contrib/setup) it all gets run for you at commit time.